app/views/declaration/items_summary.scala.html

```diff
@@ -38,17 +38,12 @@
 
     @components.error_summary(itemsErrors)
 
-    @components.heading(sectionHeader = messages("supplementary.items"))
-
-    @if(items.isEmpty) {
-        <h1 id="title" style="margin:0">@messages("supplementary.itemsAdd.title")</h1>
-    } else {
-        <h1 id="title" style="margin:0">@messages("supplementary.itemsAdd.titleWithItems", items.length)</h1>
-    }
-
-    <span class="form-hint" style="margin-bottom: 50px">@messages("supplementary.itemsAdd.title.hint")</span>
-
-
+    @components.heading(
+        sectionHeader = messages("supplementary.items"),
+        title = if(items.isEmpty) messages("supplementary.itemsAdd.title")
+        else messages("supplementary.itemsAdd.titleWithItems", items.length),
```

Comment:
I think it'll be good to extract it to variable in view

Reply:
I have reused the page title instead.